internal/resources/grafana/resource_report_test.go

```diff
@@ -38,9 +38,10 @@ func TestAccResourceReport_Multiple_Dashboards(t *testing.T) {
 					resource.TestCheckResourceAttr("grafana_report.test", "recipients.0", "some@email.com"),
 					resource.TestCheckNoResourceAttr("grafana_report.test", "recipients.1"),
 					resource.TestCheckResourceAttr("grafana_report.test", "schedule.0.frequency", "monthly"),
-					resource.TestCheckResourceAttrSet("grafana_report.test", "schedule.0.start_time"), // Date set to current time
-					resource.TestCheckResourceAttr("grafana_report.test", "schedule.0.end_time", ""),  // No end time
+					resource.TestCheckResourceAttr("grafana_report.test", "schedule.0.start_time", "2024-02-10T15:00:00-05:00"),
```

Comment:
Can't wrap my head around how this works on Grafana's side. We're sending a data with an offset to the API (-5) and also a timezone. What happens when daylight savings kicks in? A TF drift, a report sent at a different hour, or does Grafana send a correct date at read-time and all is good?

Reply:
Reports apply the timezone offset [here](https://github.com/grafana/grafana-enterprise/blob/main/src/pkg/extensions/report/scheduler/utils.go#L55).

Reply:
heh, interesting